assets/Css/ComponentsStyles.css

```diff
@@ -387,4 +387,19 @@ main {
  box-shadow: 0px 0px 5px rgb(0, 153, 255);
 }
 
+/* Responsive Image */
+
+.image-Responsive--demo{
+  width: 100%;
+  height: 100%;
+}
+.image-Responsive--bg{
+  max-width: 30rem;
```

Comment:
maybe width: 100% can take care of responsiveness, why max-width: 30rem; here?

Reply:
Yes, 100% will work but max-width is only for the size of the image.